setup.cfg

```diff
@@ -1,5 +1,21 @@
 [flake8]
-exclude = build,.git,build-pypy,.tox
+exclude =
+    .git,
+    __pycache__,
+    docs/source/conf.py,
+    docs/conf.py,
+    old,
+    build,
+    dist,
```

Comment:
This directory doesn't contain any python files

Reply:
There are, otherwise, why does he make simulations and make kits? These will be imported into this folder as temporary files and then converted into packages or simulations, such as `venv/`. And this list is provided by Flake8 on the official website, instead of just posting it by myself, and I have sorted it out.

Please refer to : https://stackoverflow.com/questions/3816522/setuptools-is-including-dist-folder-in-build

Reply:
If you think this is an issue, please fix it in setuptools itself first; there is not much point in using virtual environments for dulwich itself, so why add this here?

Reply:
Because other developers who will use the virtual environment will use it, and if it is not ignored by the project, it will be checked.

And this is the way these packages are used, so these directories or files will be generated, just like `__pycache__`.

Although there will be no impact on GitHub Actions, developers can also set this function globally like Git, but it is relatively inconvenient to `~/.gitignores_global`.

It is recommended to set it in the project, because the difference of each project is more complicated than that of `.gitignore`. You cannot set the common list as simple as `.gitignore`, and restricting the rules to projects is to generate the rules The interference is reduced and independent, so Git repositories and projects have `.gitignore` rules or flake8 exclude rules.

Reply:
I can't reproduce the issue that you link to on stackoverflow. Wouldn't that only happen if they e.g. have wildcards in MANIFEST that match the dist files?

Furthermore, this is the exclude list for *flake8*, not for source dist creation.

Reply:
Sorry, I just want to express that some directories and files will be generated when the package is made and installed. When using Flake8, MyPy, Pylint, etc. to check the directories, these directories will be checked, so these directories are added to the ignore list. E.g:

```
exclude =
    .git,
    __pycache__,
    docs/conf.py,
    old,
    build,
    dist,
    build-pypy,
    .tox,
    .eggs,
    venv
```